### PR TITLE
Fix Bug #4569

### PR DESCRIPTION
**Pull request by tgrycuk**
closes #...
After some minor changes, building the docker image works.
### Screenshots
image

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `data-test-id` are added for new elements
4. [ ] The changes are tested in Chrome/Firefox/Safari browsers and in light/dark mode
5. [ ] Your code works with the latest stable version of the core
6. [ ] I added changesets file (instructions in [contribution guide](https://github.com/saleor/saleor-dashboard/blob/main/.github/CONTRIBUTING.md)

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] app
3. [ ] attribute
4. [ ] category
5. [ ] collection
6. [ ] customer
7. [ ] giftCard
8. [ ] homePage
9. [ ] login
10. [ ] menuNavigation
11. [ ] navigation
12. [ ] orders
13. [ ] pages
14. [ ] payments
15. [ ] permissions
16. [ ] plugins
17. [ ] productType
18. [ ] products
19. [ ] sales
20. [ ] shipping
21. [ ] translations
22. [ ] variants
23. [ ] vouchers

CONTAINERS=2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM node:18-alpine as builder
+RUN apk --no-cache add bash
```
Comment (andrzejewsky) [inline Dockerfile:2]: why do you need to install bash?

**Comment by andrzejewsky** [comment]
Hey @tgrycuk thanks for contribution! Please accept CLA + add the changeset file (as patch)

File: Dockerfile (hunk 1 above)
Comment (tgrycuk) [reply Dockerfile:2]: image

Without bash this script cannot execute. And this script is executed while building the image.


image

Comment (andrzejewsky) [reply Dockerfile:2]: basically, we don't use sentry at all within the docker but the script is being executed even if it skips the sentry-related scripts. Ok let's keep the bash for now, will perhaps wok on some workaround later.

Comment (tgrycuk) [reply Dockerfile:2]: My goal was to make a docker image build. I didn't want to interfere with the script. I left the script changes for other issues.

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**